### PR TITLE
EOL Copr APIv1 and APIv2

### PR DESCRIPTION
**Pull request by xsuchy**
`bundle exec jekyll serve` does not work for me, so I had no chance to local test it how it actually looks like. Review will be appreciated.

### CODE REVIEW

**Comment by dturecek** [comment]
A [screenshot](https://dturecek.fedorapeople.org/blogpost-screenshot.png) of how the blog post looks like.

File: _posts/2020-08-04-EOL-APIv1-APIv2.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+author: Miroslav Suchý
+layout: post
+---
+During Copr history, we got [three APIs](http://frostyx.cz/posts/copr-has-a-brand-new-api). For a long time, we maintained all versions.
+
+We decided that it is time to remove the old versions. We are going to start with APiv1.  
+
+The schedule is:
+
+  * September 2020 - print warning on STDERR when you use APIv1 from python3-copr.
```
Comment (praiskup) [inline _posts/2020-08-04-EOL-APIv1-APIv2.md:12]: This is still not implemented :-(  I am not sure what is the proper way of implementing this (just doing `write(stderr, "something")` isn't a correct thing to do).  I'd like to see some historical examples in other libraries.

File: _posts/2020-08-04-EOL-APIv1-APIv2.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  * April 2021 - print warning on STDERR when you use APIv2 from python3-copr and remove APIv1 from python3-copr library
+  * September 2021 - remove APIv1 code from the frontend. I.e., Copr service will not understand APIv1 calls. Remove APIv2 from python3-copr library.
+  * April 2022 - remove APIv2 code from the frontend. I.e., Copr service will not understand APIv2 calls.
+
+I strongly encourage you to [migrate](https://python-copr.readthedocs.io/en/latest/client_v3/migration.html#migration) your scripts to [APIv3](https://python-copr.readthedocs.io/en/latest/ClientV3.html).
```
Comment (praiskup) [inline _posts/2020-08-04-EOL-APIv1-APIv2.md:17]: `s/I/We/` would sound a bit more consistent with previous sentences

**Comment by FrostyX** [comment]
We have a quite nice comparison table between APIv1 and APIv3 that might help some people during migration.
https://python-copr.readthedocs.io/en/latest/client_v3/migration.html

You might mention it somewhere.

**Comment by praiskup** [comment]
Except that I am not sure how to correctly implement the stderr warnings (logging?, write to stderr?, anything else?) looks good to me.

**Comment by xsuchy** [comment]
@FrostyX the migration is linked. I added it just a second after creating this PR.

**Comment by xsuchy** [comment]
@praiskup See https://www.idiotinside.com/2016/12/17/python-warnings-framework/ and https://docs.python.org/3.5/library/warnings.html#module-warnings
The code should be:
  ```
import warnings
  warnings.warn("This library is going to be EOLed. See...", DeprecationWarning)
```

**Pull request merged**